config/manager/kustomization.yaml

```diff
@@ -17,6 +17,9 @@ configMapGenerator:
 - envs:
   - devfile_registry.properties
   name: devfile-registry-config
+- envs:
+    - feature_flag.properties
+  name: feature-flag-config
```

Comment:
where will this feature flag be set?

Reply:
It'll be set in the [configMapGenerator](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/has/staging/kustomization.yaml#L8) for each environment overlay:

The changes will look something like this:

For Staging:
```
configMapGenerator:
  - literals:
      - GITHUB_ORG="redhat-appstudio-appdata-staging"
    name: github-config
    behavior: replace
  - literals:
      - ENVIRONMENT="staging"
    name: feature-flag-config
    behavior: replace
```

For Prod:
```
configMapGenerator:
  - literals:
    - GITHUB_ORG="redhat-appstudio-appdata"
    name: github-config
    behavior: replace
  - literals:
    - ENVIRONMENT="production"
    name: feature-flag-config
    behavior: replace
```

There will be one for Dev too.